evmbin/src/main.rs

```diff
@@ -14,7 +14,21 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Parity EVM interpreter binary.
+//! Parity EVM Interpreter Binary.
+//!
+//! ## Overview
+//!
+//! The Parity EVM interpreter binary is an additional tool in the Parity
+//! Ethereum toolchain. It is an EVM implementation for Parity Ethereum that
+//! may be used to run a standalone version of the EVM interpreter.
+//!
+//! ## Usage
+//!
+//! Build Parity Ethereum from source, then start it with:
```

Comment:
So it is not included in regular releases? That is a bit unusual so perhaps worth expanding on, like "The `evmbin` tool is not distributed with regular Parity Ethereum releases so you need to build it from source like so: …`?

Reply:
i've pushed a commit fixing this, thanks!

Reply:
Are you happy with the changes @dvdplm?